Demo/Application/Base/BraintreeDemoContainmentViewController.m

```diff
@@ -76,15 +85,19 @@ - (void)updateStatus:(NSString *)status {
 - (void)tappedStatus {
     NSLog(@"Tapped status!");
 
-    if (self.latestTokenizedPayment) {
-        NSString *nonce = self.latestTokenizedPayment.nonce;
+    if (self.latestTokenizedPayment || self.latestTokenizedPaymentString) {
```

Comment:
Can I get some context on this change? Why do we have to ways of retrieving the nonce now here?

Reply:
Yeah, so [this comment](https://github.com/braintree/braintree_ios/pull/792#discussion_r818851499) goes into a bit more details, but it has to do with some Objective-C intricacies with Swift. We have a research story for this to see if it's possible, but has not been an obvious solution yet. Since this is going into a feature branch (and not master), we plan to revisit it before merging in the feature branch. Since `BTPaymentMethodNonce` is public we also will need to find a way to force what we want to happen while allowing us to do it without breaking changes and also avoiding a mixed module (as we cannot just add a bridging header to this module because that would break our Swift Package Manager integration since it doesn't allow Obj-C + Swift to be mixed in a single module).